services/online_exam_service/.env

```
@@ -1,18 +1,29 @@
-# SERVER CONFIG
-PORT=3000
+# Server
+PORT=9003
+NODE_ENV=development
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NODE_ENV key should go before the PORT key

(UnorderedKey)